### ISSUE TITLE
[rActionBar] Bar4 shows up after leaving vehicle which has an OverrideActionBar

### ISSUE THREAD

**Issue by Pingumania** [zorker/rothui#24]
Hi,

I have the two bars on the right side disabled, but everytime I leave a vehicle which has its own spells aka the OverrideActionBar one of the right bars shows up with a slide in animation.

After a /reload the bar is gone again.

**Comment by zorker** [comment]
What do you mean by disabled? Visible on mouseover, then just hover any button? 
If you want to disable the bar use this in the config of your bar layout. 

```
frameVisibility = "hide",
```

**Comment by Pingumania** [comment]
By disabled I mean that I unchecked the two right bars in the default wow interface config. Probably wont matter?

I used rActionBar_Default

```
local bar4 = { 
  framePoint = { "RIGHT", UIParent, "RIGHT", -5, 0 },
  frameScale      = 1,
  framePadding    = 5,
  buttonWidth     = 32,
  buttonHeight    = 32,
  buttonMargin    = 5,
  numCols         = 1,
  startPoint      = "TOPRIGHT",
  fader           = fader,
}
--create
rActionBar:CreateActionBar4(A, bar4)
```

So the bar should be hidden until I mouseover it, but the bar stays visible after leaving said vehicles. 
I forgot to check if the bar will be faded again if I try to mouseover it.

Can do more testing later.

edit: I also tested this without any addons enabled and with both right bars disabled. When I click on a vehicle and the OverrideActionBar is triggered then the ObjectiveTracker moves to the left and quickly back to the right. As if I would quickly enable and disable bar4 with one difference that bar is not visible during the "moving".
